ts/components/services/ServicesSectionsList.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * A component to render a list of services organized in sections, one for each organization.
+ */
+import { Option } from "fp-ts/lib/Option";
+import I18n from "i18n-js";
+import { Button, Text, View } from "native-base";
+import React, { ComponentProps } from "react";
+import { Image, NativeScrollEvent, NativeSyntheticEvent } from "react-native";
+import { StyleSheet } from "react-native";
+import customVariables from "../../theme/variables";
+import IconFont from "../ui/IconFont";
+import ServiceList from "./ServiceList";
+import ServiceSectionListComponent from "./ServiceSectionListComponent";
+
+type AnimatedProps = {
+  animated?: {
+    onScroll: (_: NativeSyntheticEvent<NativeScrollEvent>) => void;
+    scrollEventThrottle?: number;
+  };
+  paddingForAnimation: boolean;
+  AnimatedCTAStyle?: any;
```

Comment:
Is **_AnimatedCTAStyle_** needed?
It seems to be unused

Reply:
You're right. It is removed within 3e8dd55e69564af5309be09820d64059dd82cc3c